src/Features/LanguageServer/Protocol/Handler/Configuration/DidChangeConfigurationNotificationHandler_OptionList.cs

```diff
@@ -60,6 +60,7 @@ internal partial class DidChangeConfigurationNotificationHandler
             // Code lens options
             LspOptionsStorage.LspEnableReferencesCodeLens,
             LspOptionsStorage.LspEnableTestsCodeLens,
+            LspOptionsStorage.LspFormattingSortImports,
```

Comment:
Think we need a vscode side change to add this option to our package.json, for example https://github.com/dotnet/vscode-csharp/blob/main/package.json#L1734